external/runtime/runtime.depproj

```diff
@@ -48,22 +48,6 @@
     </PackageReference>
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="@(RefAndRuntimePackageReference)" />
```

Comment:
It's shared by `corefx-runtime.depproj` and `netstandard.depproj` now. If you know how to make a single `corefx.depproj` that deploys both refs and runtimes, then I'd love to get rid of it--wasn't able to find a way.

I looked at having two `BinPlaceConfiguration` items with different metadata, but I didn't see what I could set to have them resolve refs vs. runtimes.